agreement/player_test.go

```diff
@@ -3234,19 +3234,20 @@ func TestPlayerAlwaysResynchsPinnedValue(t *testing.T) {
 }
 
 // test that ReceivedAt and ValidateAt timing information are retained in proposalStore
-// when the payloadPresent and payloadVerified events are processed, and that both timings
+// when the payloadPresent, payloadVerified, and voteVerified events are processed, and that all timings
 // are available when the ensureAction is called for the block.
-func TestPlayerRetainsReceivedValidatedAt(t *testing.T) {
+func TestPlayerRetainsReceivedValidatedAtOneSample(t *testing.T) {
 	partitiontest.PartitionTest(t)
 
 	const r = round(20239)
-	const p = period(1001)
```

Comment:
why re-doing existing tests? Shouldn't they continue to work?

Reply:
The player only records ValidatedAt for period 0